content/doc/book/installing/linux.adoc

```diff
@@ -47,9 +49,11 @@ It can be installed from the link:https://pkg.jenkins.io/debian/[`debian` apt re
 
 [source,bash]
 ----
-wget -q -O - https://pkg.jenkins.io/debian/jenkins.io.key | sudo apt-key add -
-sudo sh -c 'echo deb https://pkg.jenkins.io/debian binary/ > \
-    /etc/apt/sources.list.d/jenkins.list'
+curl -fsSL https://pkg.jenkins.io/debian-stable/jenkins.io.key | sudo tee \
+  /usr/share/keyrings/jenkins-keyring.asc > /dev/null
+sudo sh -c 'echo deb [signed-by=/usr/share/keyrings/jenkins-keyring.asc] \
+  deb https://pkg.jenkins.io/debian binary/ > \
+  /etc/apt/sources.list.d/jenkins.list'
```

Comment:
Since we're using the `tee` command in the earlier command to limit the scope of the root operation, could we use the same technique in this command:
```suggestion
echo deb [signed-by=/usr/share/keyrings/jenkins-keyring.asc] \
  https://pkg.jenkins.io/debian-stable binary/ | sudo tee \
  /etc/apt/sources.list.d/jenkins.list
```